modules/ROOT/pages/index.adoc

```diff
@@ -82,6 +82,134 @@ Monitor APIs and endpoints at scheduled intervals. Receive reports about whether
 
 See xref:api-functional-monitoring::afm-in-anypoint-platform.adoc[API Functional Monitoring in Anypoint Monitoring].
 
+== Anypoint Monitoring Features
+
+Anypoint Monitoring provides many features to Gold and Platinum subscribers. However, for more comprehensive monitoring services, Titanium subscribers receive additional charts and functionality.
+
+|===
+| Feature | Gold/Platinum Support | Titanium Support
+
+3+| *Built-in Application Dashboards*
```

Comment:
For built-in dashboards, instead of listing out every tab, I'd like to suggest talking about the 80+ out-of-the-box metrics covering the following categories: Overview, Inbound, Outbound, Performance, Failure, JVM, Infrastructure. For Titanium, there are additional advanced metrics in those categories, plus new categories for Flows, Connectors, and Application Network.

Reply:
Added.

Reply:
Sorry I changed my mind regarding word usage. Let's say "....80+ out-of-the-box charts covering metrics in the following categories..."